community/fwsnort/APKBUILD

```diff
@@ -1,19 +1,20 @@
 # Contributor: Stuart Cardall <developer@it-offshore.co.uk>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=fwsnort
-pkgver=1.6.6
+pkgver=1.6.7
 pkgrel=0
 pkgdesc="Application layer IDS/IPS by translating snort rules into iptables"
 url="http://www.cipherdyne.org/fwsnort/"
 arch="noarch"
 license="GPL"
 depends="perl perl-netaddr-ip perl-iptables-parse iptables ip6tables net-tools wget"
 install="$pkgname.post-install"
+options="!check"
```

Comment:
Does [test/test-fwsnort.pl](https://github.com/mrash/fwsnort/blob/master/test/test-fwsnort.pl) not work?